crates/storage/db/src/implementation/mdbx/mod.rs

```diff
@@ -96,12 +100,29 @@ impl DatabaseArguments {
     pub const fn new(client_version: ClientVersion) -> Self {
         Self {
             client_version,
+            geometry: Geometry {
+                size: Some(0..(4 * TERABYTE)),
+                growth_step: Some(4 * GIGABYTE as isize),
+                shrink_threshold: Some(0),
+                page_size: Some(PageSize::Set(4096)),
```

Comment:
@mattsse I can't use `default_page_size()` here because it's not a const function. `page_size` will be overridden with the value returned from `default_page_size()` where `db.database_args()` is called, which happens when the node is launched. Just want to check that this doesn't introduce any unintended side-effects in terms of tests or otherwise.

Reply:
we can make this function non-const to use the `default_page_size`, so we have the same behavior as before

Reply:
fixed, thanks